spark/src/test/scala/org/apache/spark/sql/benchmark/CometTPCDSMicroBenchmark.scala

```diff
@@ -104,15 +104,27 @@ object CometTPCDSMicroBenchmark extends CometTPCQueryBenchmarkBase {
       }
       val numRows = queryRelations.map(tableSizes.getOrElse(_, 0L)).sum
       val benchmark = new Benchmark(benchmarkName, numRows, 2, output = output)
-      benchmark.addCase(s"$name$nameSuffix") { _ =>
+      benchmark.addCase(s"$name$nameSuffix: Spark Scan + Spark Exec") { _ =>
         cometSpark.sql(queryString).noop()
       }
-      benchmark.addCase(s"$name$nameSuffix: Comet (Scan)") { _ =>
+      benchmark.addCase(s"$name$nameSuffix: Comet Scan + Spark Exec") { _ =>
         withSQLConf(CometConf.COMET_ENABLED.key -> "true") {
           cometSpark.sql(queryString).noop()
         }
       }
-      benchmark.addCase(s"$name$nameSuffix: Comet (Scan, Exec)") { _ =>
+      benchmark.addCase(s"$name$nameSuffix: Comet Scan + Comet Exec") { _ =>
```

Comment:
I'm not sure this is the PR to blow up the diff with a refactor, but we may consider standardizing this terminology (`Comet (Scan, Exec)` vs. `Comet Scan + Comet Exec`) for searching the code base for test/benchmark cases or downstream processing of the benchmark output.

Reply:
Sure, I can switch back to more consistent naming with the existing tests for this PR

Reply:
As long as the name remains consistent across commits once we start recording the benchmark results, it shouldn't matter.  
Here's an [example from Spark](https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/AggregateBenchmark.scala) btw.
But if it helps to standardize, sure.